baby-gru/src/types/privateer.d.ts

```diff
@@ -0,0 +1,23 @@
+export interface TorsionEntry {
+    sugar_1: string;
+    sugar_2: string;
+    atom_number_1: string;
+    atom_number_2: string;
+    phi: number;
+    psi: number;
+}
+
+export interface PrivateerResultsEntry {
+    svg: string;
+    wurcs: string;
```

Comment:
I would prefer to use `type` instead of `interface` here since there's no inheritance happening.